.release-please-manifest.json

```diff
@@ -1,4 +1,4 @@
 {
   "ucan": "0.1.0",
-  "ucan-key-support": "0.1.0"
```

Comment:
hoping this will return to 0.1.0 upon updates to key support?

Reply:
:pray:

Reply:
I think we should have marked the dependency bump as a breaking change :sweat:

Reply:
It's not too late; we can amend the commit on `main`....

Reply:
@cdata tried the amend.

Reply:
ah, derp, by adding `BREAKING CHANGE`.

Reply:
ok, yeppers.